sbt-plugin/src/main/scala/ch/epfl/scala/GithubDependencyGraphPlugin.scala

```diff
@@ -137,7 +145,7 @@ object GithubDependencyGraphPlugin extends AutoPlugin {
           configReport <- report.configurations
           moduleReport <- configReport.modules
           moduleRef = getReference(moduleReport.module)
-          if !moduleReport.evicted && !alreadySeen.contains(moduleRef)
+          if !moduleReport.evicted && !alreadySeen.contains(moduleRef) && !isIgnored(moduleReport.module)
```

Comment:
This will filter out the library but not its transitive dependencies. Maybe I am wrong but I think it would be quite hard to filter out all transitive dependencies of a library. To do so we would need to build the entire tree of parents and check that all roots are ignored. Unfortunately, in a module report we only have one level of parents: `moduleReport.callers`.

What about filtering out entire configs rather than dependencies? That would be easier to implement.